docs/content/concepts/01-handlers.md

```diff
@@ -0,0 +1,240 @@
++++
+title = "Request Handlers and Routing"
+slug = "handlers"
+weight = 1
++++
+
+## Introduction
+
+Request handlers are a fundamental building block of Boltzmann applications.
+They are functions which you provide to Boltzmann to **handle** incoming HTTP
+requests.
+
+<!-- more -->
+
+This document will cover how to write handler functions, how to control request
+routing, how handler responses are interpreted, and how to control these
+behaviors using attributes on your exported functions.
```

Comment:
I moved it all up into the `<!-- more -->` chunk. It might be most useful to edit our summaries when we have more of the chapters done for this section, so we can see them in context.